src/jgikbase/idmapping/util/Util.py

```diff
@@ -0,0 +1,27 @@
+'''
+
+Utility functions
+
+@author: gaprice@lbl.gov
+'''
+from typing import cast
+
+# TODO docstrings should use """
+# TODO don't add object to class declaration
+
+
+def not_none(obj: object, name: str):
+    '''
+    Check if an object is not None. In the case of a string, the string must also contain
+    non-whitespace characters.
+
+    If the object is None or a string is whitespace-only, a ValueError will be thrown.
+
+    :param obj: the object to check
+    :param name: the name of the object to use in error messages.
+    '''
+    if type(obj) == str:
```

Comment:
My suggestion here would be to call this `not_blank` and assert that obj must be a string rather than casting it. That makes the semantics much clearer to me as well ("must be a non-empty, non-whitespace string")
```py
def not_blank(string, name):
  if not isinstance(string, str) or not string.strip():
    raise ValueError(name + ' must be a non-blank string')
```

Reply:
But this is for testing general objects as well as strings.